docs/our_work/linkage.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Data Linkage Enhancement'
+summary: 'The data linkage enhancement project encompasses all things data linkage, from documenting the existing state of linkage in NHS England in the MPS handbook, to exploring better matching algorithms using Splink, to creating a quality assurance framework for data linkage.'
+origin: 'NHS Digital'
+tags: ['BEST PRACTICE', 'LINKAGE', 'PYTHON', 'QUALITY']
+---
+![Diagram representing the three current areas of the data linkage project: Quality Assurance, Better Matching Algorithm, and Data Linkage as a service.](../images/linkage_visual_abstract.png)
+
+Data linkage is an essential part of being able to use data within any organisation, but in particular the NHS. Being able to trace patients across their care journey, making underrepresented populations not be lost in the cracks, and ensuring compatibility when using several data sets for the same purpose is a pillar of why the data linkage team exists, and why we do what we do. 
+
+# **Work we do**
+
+## Data Linkage as a service
+This is the umbrella to everything else we do. We want to make linkage as easy as possible for teams within the NHS, but also help external researchers find the information about their patient cohorts that they require, and whilst the NHS number is a starting point for this, it isn't perfect and not all data sets even have NHS numbers for all patients! 
+
+## Better Matching Algorithm
+We have worked on [documenting MPS](https://digital.nhs.uk/services/personal-demographics-service/master-person-service/the-person_id-handbook), to make the current process of linking data in the NHS more transparent and easy to understand, so people know the data they're getting using it. But at the same time, we're constantly working on making it better, currently working on implementing a [probabilistic linkage model](https://www.bristol.ac.uk/media-library/sites/cmm/migrated/documents/problinkage.pdf), in order to improve linkage outcomes, and by extension, patient outcomes. 
+
+## Quality Assurance Framework
+A big part of what we do is how do we do linkage, but the underlying question is, how do we do linkage **well**. Creating a quality assurance framework for anyone within the NHS or externally to use who is wanting to link data is a big part of our goals. 
+
+
```

Comment:
Any scope for a links section as seen in other pages? It would be good if we can follow a standard format. you could link to the githubs, MPS handbook, etc.